File: Java/java2/Java5/src/woosongbit/sample1/Member.java
package woosongbit.sample1;
//회원정보를 담는 데이터 클래스
public class Member 
{
	//맴버 변수
	private int number;
	private String name;
	private String phone;
	private static int s_number=1;
	
	//생성자
	public Member(String name,String phone)
	{
		this.number=s_number++;//후치
		this.name=name;
		this.phone=phone;
	}
	//get set
	public String getName()
	{
		return name;
	}
	
	public void setPhone(String phone)
	{
		this.phone=phone;
	}
	//메서드
	public void Println()
	{
		System.out.println("번호"+number);
		System.out.println("번호"+name);
		System.out.println("번호"+phone);
	}

}

File: CppJava/Project5/Project5/Member.cpp
#include "member.h"
#include<iostream>
using namespace std;


Member::Member(int _number, const char* _name, const char _gender, const char* _phone)
{
	number = _number;
	strcpy_s(name, sizeof(name), _name);
	gender = _gender;
	strcpy_s(phone, sizeof(phone), _phone);
}


Member::~Member()
{

}


void Member::Print()
{
	cout << number << "\t\t" << name << "\t" << (gender == 'm' ? "남" : "여") << "\t" << phone << endl;
}


void Member::PrintIn()
{
	cout <<"번호 : "<< number << "\n이름 : " << name << "\n성별 : " << (gender == 'm' ? "남" : "여") << "\n전번 : " << phone << endl;
}
